src/luascript.cpp

```diff
@@ -12260,7 +12260,7 @@ int LuaScriptInterface::luaMonsterTypeIsSummonable(lua_State* L)
 	MonsterType* monsterType = getUserdata<MonsterType>(L, 1);
 	if (monsterType) {
 		if (lua_gettop(L) == 1) {
-			pushBoolean(L, monsterType->info.isAttackable);
+			pushBoolean(L, monsterType->info.isSummonable);
 		} else {
 			monsterType->info.isAttackable = getBoolean(L, 2);
```

Comment:
what about this one?

Reply:
i received some issue via whatsapp on my project [opentibiabr/OTServBR-Global](https://github.com/opentibiabr/OTServBR-Global), saying players can summon all monsters (including bosses)... so i researched on source and check why this is happened and got it just by changing this line.